pkgs/common-updater/generic-updater.nix

```diff
@@ -104,5 +111,10 @@ let
     echo "" >> ${fileForGitCommands}
   '';
 
-in
-[ updateScript pname version attrPath versionLister ignoredVersions rev-prefix odd-unstable patchlevel-unstable ]
+in {
+  name = "generic-update-script";
+  command = [ updateScript name pname version attrPath versionLister ignoredVersions rev-prefix odd-unstable patchlevel-unstable ];
+  supportedFeatures = [
+    "commit"
```

Comment:
Committing is always supported, the commit feature just means that the update script will provide the commit message on stdout, see [the docs](https://github.com/NixOS/nixpkgs/commit/c1b05442ffd6cf3cf529cad469bebe8169b156e9).

Reply:
Then the commit feature should not be enabled, because the update script does not provide the commit message on stdout.

Removed the commit feature.